### ISSUE TITLE
Can't reach the accuracy of your model

### ISSUE THREAD

**Issue by liujingqwq** [huanghoujing/EANet#28]
Hi, @huanghoujing ,
 I trained many times , use [warm up learning rate] +[triplet loss]+[id loss]+[batch size 64],  but never  achieve the accuracy of the model you provide ‘test_paper_models/PAP_ST_PS/market1501_to_duke’, Can you provide your training parameter settings? 
thanks very much.
